### DISCUSSION TITLE
Weird dash character on Mac version

### DISCUSSION THREAD

**Discussion by rodhash** [junegunn/fzf#3051] [Q&A] [answered]
Hi

I have a question about a weird dash char that I can't find where it came from. It's not impacting anything, FZF still rocks!

I'm coming from Linux and setting up this M1 mac and suddenly I noticed this weird dash after the files indicator `7/7` and it goes all the way to the end.

<img width="594" alt="image" src="https://user-images.githubusercontent.com/29671981/202093332-66e00ad2-368a-4c67-bb9d-0918dfb38762.png">

At work I have another Mac (non silicon) and didn't see this issue, so wondering if M1 chip has something to do with it .. also weird is that, fzf seems a bit slower here but this is a topic for another thread.

I have no clue where this dash char came from, I even reinstalled from homebrew and also tried installing directly from git clone.. but this kind of "dash" always stay there.

Here my zsh config:

```
# FZF config
[ -f ~/.fzf.zsh ] && source ~/.fzf.zsh

export FZF_DEFAULT_COMMAND='fd --type file'
export FZF_CTRL_T_COMMAND="$FZF_DEFAULT_COMMAND"
export FZF_DEFAULT_COMMAND="fd --type file --color=always"
export FZF_DEFAULT_OPTS="--ansi"

export FZF_COMPLETION_TRIGGER='~~'
export FZF_COMPLETION_OPTS='+c -x'
```

Is there any kind of variable or config for Mac version in order to change that?

I mainly use ZSH but apparently it also shows up on a fresh new Bash terminal.

I just run `fzf` and it shows up

Thanks.

**Comment by junegunn** [comment, marked as answer]
It's called "info separator" and was added in 0.35.0. You can customize it or remove it if you want.

See https://github.com/junegunn/fzf/releases/tag/0.35.0

**Reply by rodhash** [reply]
Thank you very much!

That lolcat thing is awesome .. now I think I'll keep it :)
